source/d3d11/d3d11_device_context.hpp

```diff
@@ -189,8 +190,24 @@ struct D3D11DeviceContext : ID3D11DeviceContext3
 	virtual void STDMETHODCALLTYPE GetHardwareProtectionState(BOOL *pHwProtectionEnable) override;
 	#pragma endregion
 
+	// local methods
+	void set_active_depthstencil(ID3D11DepthStencilView* pDepthStencilView);
+	void log_drawcall(UINT vertices);
+	void clear_drawcall_stats();
+	void log_drawcall_stats_in_tracker();
+	depth_counter_tracker& get_depth_counter_tracker();
+	
+	// fields
 	LONG _ref = 1;
+	UINT _drawcalls = 0;
+	UINT _vertices = 0;
```

Comment:
`_drawcalls` and `_vertices` appear to be unused, so can be removed.

Reply:
good catch, removed.